src/api/api.js

```diff
@@ -0,0 +1,10 @@
+const API_KEY = 'w95zbFwPDR1UrVyGdXJSbFXOSO5fcn42';
+const API_HOST = 'http://dataservice.accuweather.com/';
+const API_VERSION = 'v1';
```

Comment:
Not sure that you need a version of API in variable. 
They not changing usually so fast :)